include/urg_node/urg_node.hpp

```diff
@@ -127,7 +128,9 @@ class UrgNode : public rclcpp::Node
   int error_count_;
   int error_limit_;
   bool lockout_status_;
-
+  rclcpp::Duration system_latency_;
+  rclcpp::Duration user_latency_;
```

Comment:
this seems incorrectly named - it should be **user_time_offset_** based on the function being called